src/plugins/bookmarks/BookmarkProvider.ts

```diff
@@ -134,7 +134,7 @@ export default class BookmarkProvider extends ContactProvider {
 
    private initBookmarkContact(bookmark: RoomBookmark, service: AbstractService): IContact {
       let contact = this.createContact(bookmark.getJid());
-      contact.setNickname(bookmark.getNickname());
+      contact.getNicknameObject().setContactNickname(bookmark.getNickname());
```

Comment:
The bookmark is storing my own nickname inside the room. So I think we have to distinguish here.